integration-libs/punchout/occ/config/default-occ-punchout-config.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * SPDX-FileCopyrightText: 2025 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { OccConfig } from '@spartacus/core';
+import { PunchoutOccEndpoints } from '../model/occ-punchout.model';
+
+const punchoutOccEndpoints: PunchoutOccEndpoints = {
+  punchoutSession: 'punchout/sessions/${sId}',
+  punchoutRequisition: 'punchout/sessions/${sId}/requisition',
```

Comment:
Doesn't requisition affect a specific session?
So we could change a name to `punchoutSessionRequisition`?